src/components/tooltip/tooltip.tsx

```diff
@@ -82,6 +83,12 @@ export class Tooltip {
     @Prop({ reflect: true })
     public maxlength?: number = DEFAULT_MAX_LENGTH;
 
+    /**
+     * Decides the tooltip's location in relation to its trigger.
+     */
+    @Prop({ reflect: true })
+    public openDirection: OpenDirection = 'top';
```

Comment:
Hmm… it's a bit of a pity that we didn't include this when making the component public, because now we're changing the default behaviour to be something different than what the behaviour was before… 😣

I guess it can't be helped, and it's not likely to affect anyone, but still.

Reply:
I thought about it a bit actually, but decided to keep things simple and trackable. But since this component has been a private component and consumers have used it at their own risk, I think this change is fine. Also, it's not really a breaking change I'd say. The previous behavior was so bad anyway... So this is a feature that makes things better.

But it would have been nicer to release the component publicly, already with this feature.